mobile/src/reducers/calendar/calendar-events.reducer.ts

```diff
@@ -85,6 +91,11 @@ const createInitState = (): CalendarEventsState => {
             const now = moment();
             return event.dates.endDate.isSameOrAfter(now, 'date');
         },
+        requests: Map<string, CalendarEvent[]>(),
+        requestsPredicate: (event: CalendarEvent) => {
```

Comment:
what's the point of this predicate? We want to display all requested events, even those that a currently in the past

Reply:
Probably it was added at the very beginning in case if we need some. Going to remove it.